src/hooks/useWindowWidth.ts

```diff
@@ -0,0 +1,20 @@
+import {useEffect, useState} from 'react';
+
+const useWindowWidth = (): number => {
+  const [windowWidth, setWindowWidth] = useState<number>(0);
```

Comment:
```suggestion
  const [windowWidth, setWindowWidth] = useState<number>(window.innerWidth);
```

I think it make sense to set default value to current window width, instead of 0